Fixed #4797 : Peer review: Add one more choice to the list, and do not submit if no reason selected

Rishavgupta12345 opened the pull request:
Fixes #4797 
Peer review: Add one more choice to the list, and do not submit if no reason selected

Rishavgupta12345 commented on app/src/main/res/values/strings.xml:
```diff
@@ -510,6 +510,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="delete_helper_ask_reason_copyright_press_photo">Press photo</string>
   <string name="delete_helper_ask_reason_copyright_internet_photo">Random photo from internet</string>
   <string name="delete_helper_ask_reason_copyright_logo">Logo</string>
+  <string name="delete_helper_ask_reason_copyright_breach_of_freedom_of_panorama_rules">Breach of Freedom of Panorama rules</string>
```
done 😎 thanks

nicolas-raoul reviewed (commented):
Thanks a lot! :-)

For UI pull requests, could you please send screenshots showing the results?
Here is what I see:
![adb](https://user-images.githubusercontent.com/99590/154791833-3058bd4b-5f42-45b9-941a-741767779272.png)

Rishavgupta12345 commented:
> Thanks a lot! :-)
> 
> For UI pull requests, could you please send screenshots showing the results? Here is what I see: ![adb](https://user-images.githubusercontent.com/99590/154791833-3058bd4b-5f42-45b9-941a-741767779272.png)

I updated the copyright option and added reason 4 as No freedom of panorama which is working fine , but the scope option is also adding reason 4 as null , therefore this problem is happening ....  I will solve it soon

Rishavgupta12345 commented:
@nicolas-raoul  I updated the copyright option and added reason 4 as No freedom of panorama which is working fine , but the (out of scope) option is also adding reason 4 as null , therefore this problem is happening .... 

can you please guide me how to solve this , I have tried but couldn't figure out .
for me the best option will be if I also add a Reason 4 for(out of scope), as you wish.

nicolas-raoul commented:
Do you mean that in the current master, each of the step must have the exact same number of possible reasons?

That does not sound ideal, even though I guess that saves some lines of code.

How about designing a system that can take different numbers of reasons at each step?

nicolas-raoul commented:
Could you please rebase on master? A related area has been changed in master. Thanks!

Rishavgupta12345 commented:
> Could you please rebase on master? A related area has been changed in master. Thanks!

yes done ,

Rishavgupta12345 commented:
@nicolas-raoul  for now i am unable to remove the REASON 4 in out of scope option . I have tried many ways but it is not happenign . what should I do now ?

nicolas-raoul commented:
How about designing a system that can take different numbers of reasons at each step?

Or feel free to unassign this issue and work on a different one if you prefer.
